Gemfile.lock

```diff
@@ -0,0 +1,88 @@
+PATH
```

Comment:
Gemfile.lock for gems should be in .gitignore, odd that it was not there from the beginning

Reply:
@Robgra13 looks like you haven't deleted it